python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -562,7 +562,7 @@ MlirModule synthesizeKernel(const std::string &name, MlirModule module,
   pm.addNestedPass<func::FuncOp>(cudaq::opt::createClassicalMemToReg());
   pm.addNestedPass<func::FuncOp>(createCanonicalizerPass());
   pm.addNestedPass<func::FuncOp>(cudaq::opt::createLoopNormalize());
-  pm.addNestedPass<func::FuncOp>(cudaq::opt::createLoopUnroll());
+  pm.addPass(cudaq::opt::createLoopUnroll());
```

Comment:
Why would we want to run normalization and unrolling at different levels? Seems better if we do not.

Reply:
it only works as a non-nested pass, this was giving me an errror

Reply:
updated both to run at module level, thanks!